app.js

```diff
@@ -0,0 +1,47 @@
+
+const express = require('express');
+const app     = express();
```

Comment:
Try to put all the requires at the top and separate them form the res. It's cleaner.